test/Table_Tests/src/IO/Excel_Spec.enso

```diff
@@ -73,6 +73,41 @@ spec_fmt suite_builder header file read_method sheet_count=5 =
             t_3.at 'B' . to_vector . should_equal [Nothing, 30]
             t_3.at 'C' . to_vector . should_equal [43.2, 54]
 
+        group_builder.specify "should let you read all sheets from xlsx" <|
+            wb = read_method file
+
+            ## Read all the sheets as single rows.
+            single_table = wb.read_many
+            single_table.row_count . should_equal 25
+            single_table.column_names . should_equal ["SheetName", "Name", "Quantity", "Price", "A", "B", "C", "D", "E", "Student Name", "Enrolment Date", "Item", "Price 1"]
+            warnings = Warning.get_all single_table . map _.value
+            warnings.length . should_equal 2
+            warnings.should_contain Empty_Sheet.Error
+
+            ## Read all the sheets as tables.
+            table_of_tables = wb.read_many return=..Table_Of_Tables
+            table_of_tables.row_count . should_equal 5
+            table_of_tables.column_names . should_equal ["SheetName", "Table"]
+            table_of_tables.at "SheetName" . to_vector . should_equal ["Simple", "Strange Dimensions", "Dates", "Empty", "Duplicate Columns"]
+            Warning.get_all table_of_tables . map _.value . length . should_equal 2
```

Comment:
Checking that there is some warning but it can be anything. I'd definitely prefer to rely on our existing helpers from the Test suite that are meant for this, e.g. `Problems.expect_warning`